skins/laika/templates/page_layouts/supporters.html.twig

```diff
@@ -1,6 +1,8 @@
 {% extends 'Base_Layout.html.twig' %}
 {% set title = site_metadata.page_titles[page_id] %}
 
+{% block title %}{$ page_title( page_id ) $}{% endblock %}
```

Comment:
Maybe it should be?
```javascript
{% block title %}{$ page_title( supporters_page_title ) $}{% endblock %}
```

Reply:
I think the language key is `hall-of-fame`, but it works fine as it is, and it's consistent with the other static pages so lets leave it as is.